config/initializers/cors.rb

```diff
@@ -0,0 +1,17 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin Ajax requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins 'http://127.0.0.1:3000' # Replace this with the domain you want to allow
+
+    resource '*',
+      headers: :any,
+      methods: [:get, :post, :put, :patch, :delete, :options, :head],
+      credentials: true
+  end
+end
```

Comment:
- [ ] Great job working on the cors as requested, however your 2 applications should be able to communicate with each other. The frontend is currently unable to display the greeting stored on the backend. Kindly fix this.

Video demo
[cinnamon-2023-12-04T234237+0200.webm](https://github.com/MenendezON/hello-world-backend/assets/81932478/3e474890-eb14-4026-932d-8420d59e9972)